controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -80,13 +80,24 @@ const (
 	hcoVersionName    = "operator"
 	secondaryCRPrefix = "hco-controlled-cr-"
 	apiServerCRPrefix = "api-server-cr-"
+	configMapCRPrefix = "cm-policy-cr-"
 
 	// These group are no longer supported. Use these constants to remove unused resources
 	v2vGroup = "v2v.kubevirt.io"
 
 	requestedStatusKey = "requested status"
 )
 
+// TriggeredBy informs which resource has triggered the Reconcile loop
+type TriggeredBy int8
+
+const (
+	Hco TriggeredBy = iota
+	Api
+	Cm
+	Other
+)
+
```

Comment:
please use more declarative names, and please don't expose these const (they are not needed outside of this package):

For example:
```suggestion
const (
	triggeredByHCO TriggeredBy = iota
	triggeredByAPI
	triggeredByCM
	triggeredByOther
)
```

Reply:
Done.